### ISSUE TITLE
Aligning on behavior of prefetch

### ISSUE THREAD

**Issue by noamr** [mozilla/standards-positions#723]
## Request for Mozilla Position on changes to an Existing Web Specification

Mozilla introduced prefetch years ago... Now trying to align it across browsers. The proposal is very similar to what Mozilla already does, with a couple of changes:

* The `Accept` header for prefetch is defined as the same one that is sent with frame navigation. This prevents double-prefetches with `Vary: Accept`
* Instead of the custom `X-Moz: Prefetch` and `Purpose: Prefetch` we send today, align on `Sec-Purpose: Prefetch`

* Specification Title: <link rel=prefetch>
* Specification or proposal URL:  https://github.com/whatwg/html/pull/8111
* Caniuse.com URL (optional): https://caniuse.com/link-rel-prefetch
* Bugzilla URL (optional): https://bugzilla.mozilla.org/show_bug.cgi?id=1788167
* Mozillians who can provide input (optional):

**Comment by martinthomson** [comment]
@bgrins, @mystor, this looks reasonable to me.  WDYT?

**Comment by smaug----** [comment]
Looks reasonable to me too.

**Comment by noamr** [comment]
Thanks! I wonder what's your view on the related CSP change: https://github.com/w3c/webappsec-csp/pull/582
Tl;dr: instead of having a custom `prefetch-src`, we use `default-src` or the least restrictive directive in the policy. This allows using resource hints and exfiltration protection (e.g. `default-src 'none'`) together.

**Comment by zcorpan** [comment]
I've discussed with @valenting , we think this looks reasonable. Will label as "positive", no need for a dashboard entry.
